pkg/app/sensor/app.go

```diff
@@ -489,8 +526,14 @@ func readCommandFile() (command.StartMonitor, error) {
 	}
 
 	if err := json.Unmarshal([]byte(jsonCmd), &cmd); err != nil {
-		return cmd, err
+		return cmd, fmt.Errorf("could not decode command: %w", err)
 	}
 
 	return cmd, nil
 }
+
+func writeJSON(v interface{}, w io.Writer) {
+	if err := json.NewEncoder(w).Encode(v); err != nil {
```

Comment:
need to disable the default html escaping for the json encoder object

Reply:
Done!